packages/open-truss/src/components/index.ts

```diff
@@ -0,0 +1 @@
+export { default as OTBar } from './OTBar'
```

Comment:
To avoid name collisions with components developed in applications, I prefixed OT components with OT. What do folks think of this?

Reply:
Great idea